tests/conftest.py

```diff
@@ -1726,7 +1729,10 @@ def bucket_factory_fixture(request, mcg_obj):
         'cli': CLIBucket
     }
 
-    def _create_buckets(amount=1, interface='S3', *args, **kwargs):
+    def _create_buckets(
```

Comment:
make sure that this function usages are not affected

Reply:
Doesn't seem like we ran into any unknown bugs/issues in the run, so I believe it is safe 👍